src/stories/components/checkbox/Checkbox.stories.tsx

```diff
@@ -0,0 +1,46 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import Checkbox from '../../../components/checkbox/Checkbox';
+
+const meta: Meta<typeof Checkbox> = {
```

Comment:
nit: if you want, you can make the component interactive, so you can click in the component in the story and see what happens. You an see [the radio button story](https://github.com/internxt/ui/blob/master/src/stories/components/radio-button/RadioButton.stories.ts#L6) to get an example.

Reply:
Done